tests/Microsoft.Identity.Test.LabInfrastructure/KeyVaultSecretsProvider.cs

```diff
@@ -132,7 +132,7 @@ private async Task<string> AuthenticationCallbackAsync(string authority, string
                 confidentialApp = ConfidentialClientApplicationBuilder
                     .Create(KeyVaultConfidentialClientId)
                     .WithAuthority(new Uri(authority), true)
-                    .WithClientSecret(_config.KeyVaultSecret)
+                    .WithClientSecret("haFC7VUF61aWjhAH/Oa+11CpmzzKRxgDPth41THkO04=")
```

Comment:
Erm, I don't think we should be publishing this. Please invalidate it and use a config.

Reply:
that was test code. the secret is no longer valid

Reply:
I also rolled back the change